docs/pages/build-reference/ios-capabilities.mdx

```diff
@@ -14,7 +14,7 @@ When you make a change to your iOS entitlements, this change needs to be updated
 
 In bare workflow apps, the entitlements are read from your __ios/**/*.entitlements__ file.
 
-In the managed workflow, the entitlements are read from the introspected Expo config. You can see what your introspected config looks like by running `npx expo config --type introspect` in your project, then look for the `ios.entitlements` object for the results.
+In the managed workflow, the entitlements are read from the introspected Expo config. To edit them, see the [`ios.entitlements`](/versions/latest/config/app/#entitlements) field in your app config file. You can see what your introspected config looks like by running `npx expo config --type introspect` in your project, then look for the `ios.entitlements` object for the results.
```

Comment:
```suggestion
In the managed workflow, the entitlements are read from the introspected app config. To edit them, see the [`ios.entitlements`](/versions/latest/config/app/#entitlements) field in your app config file. You can see your introspected config by running `npx expo config --type introspect` in your project and then look for the `ios.entitlements` object for the results.
```

Reply:
Unrelated to this PR: Small suggestions to improve the context (replace Expo config with app config) and correct use of article